lib/core/presentation/pages/event/guest_event_detail_page/widgets/guest_event_poap_offer_item.dart

```diff
@@ -209,10 +210,18 @@ class GuestEventPoapOfferItemState extends State<GuestEventPoapOfferItemView>
                       builder: (context, state) {
                         final claiming = state.claiming;
                         final hasClaimed = state.claimed;
+                        final isChecking = state.checking;
                         final claimable = state.policy != null
                             ? state.policy?.result?.boolean ?? false
                             : !hasClaimed;
 
+                        if (isChecking) {
+                          return SizedBox(
```

Comment:
You should check it directly using `state.checking`. Don't create unnecessary variables that can be used directly